app/Http/Controllers/ContactUsController.php

```diff
@@ -8,6 +8,6 @@
 class ContactUsController extends Controller
 {
     public function index() {
-        return Inertia::render('ContactUs');
+        return Inertia::render('Contacts/ContactUsIndex');
```

Comment:
your version
`Contacts/ContactUsIndex`

should be:
`ContactUs/ContactUsIndex`